### ISSUE TITLE
Volkov Crashes The Game when Garrison/Fire

### ISSUE THREAD

**Issue by G0nTar** [MlemandPurrs/rapluslite#2]
Simple, n' easy to fix, **Volkov** Crashes Everyones Game when Garrisoned inside a building or vehicle that allows him to shoot from the inside.

He doesn't have a Garrison fire muzzle flash and Neither him nor OpenRA likes that

Here's the log in case the resume i did wasn't enough :)

OpenRA engine version release-20200202
Red Alert Plus (Lite) mod version v1.3.5
on map aaaf8b9ae948dfcea1fff4913edd8848bc44fe9d (Rapa Nui by American Blunt).
Date: 2020-04-09 23:31:02Z
Operating System: Windows (Microsoft Windows NT 6.1.7601 Service Pack 1)
Runtime Version: .NET CLR 4.0.30319.42000
Exception of type `System.InvalidOperationException`: Unit `volk` does not have a sequence named `garrison-muzzle`
   en OpenRA.Graphics.SequenceProvider.GetSequence(String unitName, String sequenceName)
   en OpenRA.Graphics.Animation.PlaySequence(String sequenceName)
   en OpenRA.Graphics.Animation.PlayThen(String sequenceName, Action after)
   en OpenRA.Mods.Common.Traits.AttackGarrisoned.DoAttack(Actor self, Target target)
   en OpenRA.Mods.Common.Traits.AttackFollow.Tick(Actor self)
   en OpenRA.Mods.Common.Traits.AttackGarrisoned.Tick(Actor self)
   en OpenRA.WorldUtils.DoTimed[T](IEnumerable`1 e, Action`1 a, String text)
   en OpenRA.World.Tick()
   en OpenRA.Game.InnerLogicTick(OrderManager orderManager)
   en OpenRA.Game.LogicTick()
   en OpenRA.Game.Loop()
   en OpenRA.Game.Run()
   en OpenRA.Game.InitializeAndRun(String[] args)
   en OpenRA.WindowsLauncher.RunGame(String[] args)

Basically, Volkov wants a Muzzle Flash when inside vehicles n' buildings like a pillbox or mobile bunker, etc (it happend when he tryed to enter a pillbox, and again later when somebody tryed to get him inside a Fortress from RA2)

Hope this helps! Great Mod by the way, i would twean with faction passive bonuses a bit, but otherwise really cool

[exception-2020-04-09T233102Z.log](https://github.com/MlemandPurrs/rapluslite/files/4459105/exception-2020-04-09T233102Z.log)
)

**Comment by MlemandPurrs** [comment]
sorry that this ctd got into release, i must have overlooked this one. i shall hotfix this as soon as possible.
